src/content/docs/apm/new-relic-apm/getting-started/get-started-logs-context.mdx

```diff
@@ -0,0 +1,204 @@
+
+## Ensure data privacy [#data-privacy]
+
+Our log management service automatically masks number patterns that appear to be for items such as credit cards or Social Security numbers. For more information, see our [security documentation](/docs/logs/get-started/new-relics-log-management-security-privacy) for log management.
+
+You can also use our obfuscation options to hash or mask sensitive data in your logs. This is critical, for example:
+
+* If it is impractical or impossible to split sensitive data into separate accounts
+* If you want data to be searchable and coherent without sensitive information exposed
+* If you want to restrict query capabilities for sensitive data for users without those access rights
+
+For more information, see our [obfuscation documentation](/docs/logs/ui-data/obfuscation-ui/). You can create, read, update, and delete obfuscation expressions and rules via the New Relic One UI or NerdGraph, our New Relic GraphQL explorer.
+
+## Manage your ingest limits [#ingest]
+
+Using telemetry in context functionality will increase your data ingest. Depending on your account's pricing model, this may have an impact on your ingest limits and billing.
+
+<Callout variant="caution">
+    If you want to use our telemetry in context solution and you already have an existing log forwarding solution, you must disable it. Otherwise you will be sending duplicate lines, which results in double billing. For more information, follow standard procedures to disable your [specific log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic#log-forwarding).
+</Callout>
+
+**Example:** Your engineering team is troubleshooting a problem with your app, so you want to temporarily raise the limits to maximum data collection in your agent's config file. (By default, the maximum number of logs sampled by your APM agent is approximately 10,000 per minute.) But if you leave higher limits running for several days, this could result in a lot of unnecessary data and wasted money.
+
+To avoid any surprises, we recommend that you use [NRQL queries](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#data-queries) to create [alert conditions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#alerts) to keep track of your ingest limits. For example:
+
+<CollapserGroup>
+  
+  <Collapser
+    id="limits-ui"
+    title="View data limits and usage in UI"
+  >
+
+  To [review your data limits in the UI](/docs/data-apis/manage-data/view-system-limits): From the [account dropdown](/docs/glossary/glossary/#account-dropdown) in the New Relic One UI, click **Manage your data > Limits**. Or, to view your usage and estimated cost to date, click **View your usage**.
+
+  ![View data and usage from UI](./images/view-limits.png "View data and usage from UI")
+  
+  </Collapser>
+
+  <Collapser
+    id="query-estimate"
+    title="NRQL query: estimated cost example"
```

Comment:
Capitalize after ":"?

Reply:
I've seen it both ways across our docs :-(